packages/styling/src/styles/theme.ts

```diff
@@ -60,11 +62,12 @@ export function loadTheme(theme: IPartialTheme): ITheme {
  * Creates a custom theme definition which can be used with the Customizer.
  */
 export function createTheme(theme: IPartialTheme): ITheme {
-  let newPalette = { ...DefaultPalette, ...theme.palette };
 
-  if (!theme.palette || !theme.palette.accent) {
-    newPalette.accent = newPalette.themePrimary;
-  }
+  let newPalette = { ...DefaultPalette };
+  newPalette.accent = newPalette.themePrimary;
+  newPalette.themePrimaryTransparent = _makeTransparentColor(newPalette.themePrimary, 0.1);
+
+  newPalette = { ...newPalette, ...theme.palette };
```

Comment:
can you separate these 4 lines out to another function, _makePalette(theme: IPartialTheme)